astroquery/alma/tests/test_alma.py

```diff
@@ -518,8 +522,8 @@ def run_sync(self, uid):
     alma._datalink = MockDataLinkService()
     result = alma.get_data_info(uids='uid://A001/X12a3/Xe9', expand_tarfiles=True)
 
-    # Entire expanded structure is 61 links long.
-    assert len(result) == 61
+    # Entire expanded structure is 19 links long.
```

Comment:
Is this something that will change?  If so, add the date in a comment.  If not, it's fine to leave this as-is - but it would be good to leave some note about what to do if the number changes.

Reply:
the date is in the commit, so I left this as is for now. If it starts failing we can think about modifications.